sql/core/src/test/scala/org/apache/spark/sql/execution/command/CharVarcharDDLTestBase.scala

```diff
@@ -97,6 +99,86 @@ trait CharVarcharDDLTestBase extends QueryTest with SQLTestUtils {
       assert(v1 || v2)
     }
   }
+
+  def checkRaiseErrorApplied(df: DataFrame): Unit = {
+    val re = df.queryExecution.optimizedPlan.asInstanceOf[Project].projectList.head.collect {
+      case _: RaiseError => true
+    }
+    assert(re.nonEmpty, "should apply raise_error to output")
+  }
+
+  test("create table w/ location and fit length values") {
```

Comment:
Nit: these tests are not DDL and would be better to put in `CharVarcharTestSuite` for testing padding and length check.